woocommerce/functions.php

```diff
@@ -28,6 +28,10 @@ function siteorigin_north_woocommerce_enqueue_styles( $styles ){
 		'media' => 'all'
 	);
 
+	if( 'is_woocommerce' && (siteorigin_setting( 'responsive_disable' ) == true) ) {
```

Comment:
Shouldn't this be a function call to is_woocommerce? Also, it shouldn't be neccessary to write `(siteorigin_setting( 'responsive_disable' ) == true)`, just `siteorigin_setting( 'responsive_disable' )` should do.